yapf/yapflib/reformatter.py

```diff
@@ -394,6 +398,172 @@ def _AlignTrailingComments(final_lines):
       final_lines_index += 1
 
 
+def _AlignAssignment(final_lines):
```

Comment:
This function is _very_ long. Please split it up into smaller functions so that it's more readable.